src/features/unified-share-modal/ContactsField.js

```diff
@@ -87,7 +92,7 @@ class ContactsField extends React.Component<Props, State> {
                 fullContacts = this.addSuggestedContacts(fullContacts);
             }
 
-            return fullContacts.map<Object>(({ email, id, isExternalUser, name, type }) => ({
+            return fullContacts.map<any>(({ email, id, isExternalUser, name, type }) => ({
```

Comment:
was there a reason this was changed to 'any'?

Reply:
this was a carry over from the old PR. `Object` doesn't have a strong meaning in flow. 

https://github.com/facebook/flow/issues/7291

The correct type here would be whatever object results from this `.map` call, which wasn't typed earlier. That is part of the TODO directly above.